privacyidea/lib/resolvers/SQLIdResolver.py

```diff
@@ -351,6 +351,13 @@ def getUserInfo(self, userId):
             log.error("Could not get the userinformation: {0!r}".format(exx))
 
         return userinfo
+    
+    def _get_userid_filter(self, userId):
+        column = getattr(self.TABLE, self.map.get("userid"))
+        if isinstance(column.type, Integer):
+            return column == userId
+        else:
+            return column.like(userId)
```

Comment:
Thanks a lot for the pull request.

All tests now only run through line 358. Can you please add a test, that also hits line 360.
I think this can be achieved by adding a configurion with a mapping of the userid to e.g. the username
(see https://github.com/privacyidea/privacyidea/blob/master/tests/test_lib_resolver.py#L154)